app/jobs/ingest_yaml_job.rb

```diff
@@ -107,6 +108,10 @@ def ingest_files(parent: nil, resource: nil, files: [])
       attach_files_to_work(resource, @file_sets) if @file_sets.any?
     end
 
+    def ingest_ocr(actor, f)
+      actor.create_content(ocr_file(f), "extracted_text") if ocr_file(f)
```

Comment:
BTW the responsibility for closing the File passed to create_content should be carefully noted.